lightwood/api/predictor.py

```diff
@@ -139,10 +139,25 @@ def predict(self, data: pd.DataFrame, args: Dict[str, object] = {}) -> pd.DataFr
     def save(self, file_path: str) -> None:
         """
         With a provided file path, saves the Predictor instance for later use.
+        :param file_path: Location to store your Predictor Instance.
+        :returns: Saves Predictor instance.
+        """
+        with open(file_path, "wb") as fp:
+            dill.dump(self, fp)
+
+    def export(self, file_path: str, json_ai_code: str) -> None:
+        """
+        With a provided file path, saves the Predictor instance for later use.
 
         :param file_path: Location to store your Predictor Instance.
+        :param json_ai_code: The code generated by the user's specification (optional)
```

Comment:
Minor nitpick: right now `json_ai_code` is not optional 😁

Reply:
Haha sorry! I took it out BC I was doing a new method entirely so I didn’t know if having that was pointless anymore. I’ll put those back in and send another PR this morning!!

Reply:
🤦‍♀️ oops! fixing that now